migrations/frontend/1675367314_add_results_to_permission_sync_jobs/up.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE permission_sync_jobs ADD COLUMN IF NOT EXISTS permissions_added Integer;
+ALTER TABLE permission_sync_jobs ADD COLUMN IF NOT EXISTS permissions_removed Integer;
+ALTER TABLE permission_sync_jobs ADD COLUMN IF NOT EXISTS permissions_found Integer;
```

Comment:
Mr. Nit Pick back in town:
```suggestion
ALTER TABLE permission_sync_jobs ADD COLUMN IF NOT EXISTS permissions_added integer;
ALTER TABLE permission_sync_jobs ADD COLUMN IF NOT EXISTS permissions_removed integer;
ALTER TABLE permission_sync_jobs ADD COLUMN IF NOT EXISTS permissions_found integer;
```

Reply:
updated